security/keycloak-oidc-client/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,34 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
```

Comment:
I don't think we need `docker/Dockerfile.*` files

Reply:
Deleted